### PR TITLE
Added Images in AUGSD team page and added registration page

### PR DESCRIPTION
**Pull request by naresh-golla-design**

### CODE REVIEW

**Comment by Kailash0311** [comment]
Thanks @naresh-golla-design , is this PR tested thoroughly locally and working as expected?

**Comment by naresh-golla-design** [comment]
> Thanks @naresh-golla-design , is this PR tested thoroughly locally and working as expected?

Yes. It is tested and working.

**Review by Kailash0311 — APPROVED** [review]

File: routes/admin/portals/control/registration.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+let express = require("express");
```
Comment (sohailrajdev97) [inline routes/admin/portals/control/registration.js:1]: This is a public facing endpoint. This should not be present in `routes/admin/portals/control`

**Comment by sohailrajdev97** [comment]
Ideally, we should move away from checking-in non website related static content (PDFs etc.) into source control. 

Not needed in this PR, can be taken up separately.

File: routes/admin/portals/control/registration.js (hunk 1 above)
Comment (sohailrajdev97) [reply routes/admin/portals/control/registration.js:1]: Also, why was this change needed? We are not using this router anywhere.

**Review by sohailrajdev97 — CHANGES_REQUESTED** [review]

**Comment by sohailrajdev97** [comment]
Please run these commands locally to fix linting

1.`npm install -g grunt-cli`
2. `grunt fix`

File: routes/admin/portals/control/registration.js (hunk 1 above)
Comment (sohailrajdev97) [reply routes/admin/portals/control/registration.js:1]: As discussed offline, please remove this file in a follow-up PR.

**Review by sohailrajdev97 — APPROVED** [review]

**Pull request merged**